packages/veritone-widgets/src/widgets/EnginePicker/SelectionInfoPanel.js

```diff
@@ -0,0 +1,121 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import { objectOf, any, func, string } from 'prop-types';
+
+import * as multipleEngineSelectionModule from '../../redux/modules/multipleEngineSelection';
+import widget from '../../shared/widget';
+
+import Typography from '@material-ui/core/Typography';
+import Paper from '@material-ui/core/Paper';
+import List from '@material-ui/core/List';
+import ListItem from '@material-ui/core/ListItem';
+import ListItemIcon from '@material-ui/core/ListItemIcon';
+import StarIcon from '@material-ui/icons/Star';
+import Grid from '@material-ui/core/Grid';
+import DeleteIcon from '@material-ui/icons/Delete';
+import IconButton from '@material-ui/core/IconButton';
+
+import { SelectionButton } from 'veritone-react-common';
+
+import styles from './styles.scss';
+
+@connect((state, { ids }) => {
+  return {
+    selected: multipleEngineSelectionModule.selectedEngines(state)
+  };
+})
+class SelectionInfoPanel extends React.PureComponent {
+  static propTypes = {
+    selected: objectOf(any),
+    toggleEngine: func,
+    selectSpecial: func,
+    selectSpecialLabel: string,
+    selectSpecialId: string
+  };
+
+  static defaultProps = {
+    selectSpecialLabel: 'Baseline'
+  };
+
+  getRemoveHandler = engineId => () => {
+    if (engineId === this.props.selectSpecialId) {
+      this.props.selectSpecial(undefined);
+    }
+    this.props.toggleEngine({ rowId: engineId });
+  };
+
+  getSelectionHandler = engineId => () => this.props.selectSpecial(engineId);
+
+  render() {
+    return (
+      <Paper elevation={4} style={{ height: '100%', minHeight: '300px' }}>
+        <div className={styles.pickerTitle}>
+          <Typography variant="headline">
+            Selected Engines{' '}
+            <Typography
+              variant="subheading"
+              style={{ display: 'inline-block' }}
+            >
+              {Object.keys(this.props.selected).length} / 6
```

Comment:
hard coded values scare me. can we put it in a variable that helps to make it clear what it's for? additionally, is there value in making it a prop?

Reply:
Oops, yeah I meant to make that a prop.